bcrembed/__main__.py

```diff
@@ -83,22 +83,22 @@ def antiberty(inpath: str, colname: str, outpath: str):
     stdout.print(f"Saved embedding at {outpath}")
 
 @app.command()
-def antiberta2(inpath: str, colname: str, outpath: str):
+def antiberta2(inpath: str, sequence_input: str, outpath: str):
     """
     Embeds sequences using the antiBERTa2 RoFormer model.
 
     Args:
         inpath (str): The path to the input file. The file should be in AIRR format.
-        colname (str): The name of the column in the input file that contains the sequences to be embedded.
+        sequence_input (str): Input sequences (H for heavy chain, L for light chain, HL for heavy and light concatenated). 
         outpath (str): The path where the embeddings will be saved.
 
     Note:
         This function prints the size of the model used for embedding, the batch number during the embedding process, 
         and the time taken for the embedding.
     """
     device = 'cuda' if torch.cuda.is_available() else 'cpu'
-    dat = pivot_airr(inpath)
-    X = dat.loc[:,colname]
+    dat = process_airr(inpath, sequence_input)
+    X = dat.loc[:, "sequence_vdj_aa"]
```

Comment:
I would still let the user specify the column name where the translation should be taken from, but set `sequence_vdj_aa` as the default value